presentation/src/main/java/com/android254/presentation/home/viewmodel/HomeViewModel.kt

```diff
@@ -111,7 +109,9 @@ class HomeViewModel @Inject constructor(
                 format = it.sessionFormat,
                 startDate = it.startDateTime,
                 endDate = it.endDateTime,
-                remoteId = it.remote_id
+                remoteId = it.remote_id,
+                isService = it.isServiceSession,
+                sessionImage = it.sessionImage ?: ""
```

Comment:
`it.sessionImage ?: "" ` -> `it.sessionImage.orEmpty()` but ideally this should happen in the data layer.